Tests/Runner/StateFileTest.php

```diff
@@ -563,6 +620,101 @@ public function testPersisStatePassOtherExceptions()
         $stateFile->stashState(['key' => 'fooBar']);
         $this->expectException(ClientException::class);
         $this->expectExceptionMessage("Test");
-        $stateFile->persistState();
+        $stateFile->persistState(new InputTableStateList([]));
+    }
+
+
+    public function testPersistStateStoresInputTablesState()
+    {
+        $sapiStub = self::getMockBuilder(Client::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $sapiStub->expects(self::once())
+            ->method('apiPut')
+            ->with(
+                $this->equalTo("storage/components/docker-demo/configs/config-id"),
+                $this->callback(function ($argument) {
+                    self::assertArrayHasKey('state', $argument);
+                    $data = \GuzzleHttp\json_decode($argument['state'], true);
+                    self::assertArrayHasKey(StateFile::COMPONENT_NAMESPACE_PREFIX, $data);
+                    self::assertArrayHasKey(StateFile::STORAGE_NAMESPACE_PREFIX, $data);
+                    self::assertArrayHasKey(StateFile::INPUT_NAMESPACE_PREFIX, $data[StateFile::STORAGE_NAMESPACE_PREFIX]);
+                    self::assertArrayHasKey(StateFile::TABLES_NAMESPACE_PREFIX, $data[StateFile::STORAGE_NAMESPACE_PREFIX][StateFile::INPUT_NAMESPACE_PREFIX]);
+                    $storedState = $data[StateFile::STORAGE_NAMESPACE_PREFIX][StateFile::INPUT_NAMESPACE_PREFIX][StateFile::TABLES_NAMESPACE_PREFIX];
+                    self::assertCount(1, $storedState);
+                    self::assertEquals([
+                        'source' => 'in.c-main.test',
+                        'lastImportDate' => 'today'
+                    ], $storedState[0]);
```

Comment:
nesly by vsechny ty asserty nahradit jednim asserequals ? v tom poli neni nic dynamickyho ne? myslim, ze by to bylo trosku prehlednejsi

Reply:
není no, mám to tak myslíš udělat u všech testů v `StateFileTest`?

Reply:
no u vsech to prave myslim nejde

Reply:
jj, nejde, ale myslel jsem u všech, *kde to jde*, i nad rámec změn v tomhle PR

Reply:
nechal bych to lezet